examples/mobile/UIComponents/components/controllers/radio.js

```diff
@@ -5,18 +5,17 @@
 	 * radioresult - Indicator for active radio
 	 */
 	var page = document.getElementById("radio-demo"),
-		radios = document.querySelectorAll("input[name='radio-choice']"),
-		radioresult = document.querySelector((".radio-result")),
+		radios = document.querySelectorAll("input[name='radio-group']"),
+		radiosTexts = document.querySelectorAll((".radio-text")),
 		idx;
 
 	/**
 	 * Updates text for an selected radio
 	 * @param {HTMLElement} target
 	 */
 	function setRadioresultFromTarget(target) {
-		if (target.checked) {
-			radioresult.innerHTML = "The Active Radio is " + target.id;
-		}
+		radiosTexts[target.id].innerHTML = "Radio on";
+		radiosTexts[(parseInt(target.id, 10) + 1) % 2].innerHTML = "Radio off";
```

Comment:
You added `id` for  these elements. I think is better to use it. I think simpler is better, this is only sample.

Reply:
simplified